classes/PodsAdmin.php

```diff
@@ -2090,7 +2090,7 @@ public function admin_components () {
             if ( !empty( $component_data[ 'category' ] ) ) {
                 $category_url = pods_query_arg( array( 'view' => sanitize_title( $component_data[ 'category' ] ), 'pg' => '', 'page' => $_GET[ 'page' ] ) );
 
-                $component_data[ 'category' ] = '<a href="' . esc_url( $category_url ) . '">' . $component_data[ 'category' ] . '</a>';
+                $component_data[ 'category' ] = '<a href="' . esc_url( $category_url ) . '">' . __( $component_data[ 'category' ], 'pods' ) . '</a>';
```

Comment:
I see, you are right, `Category` string is used elsewhere and that is why it works. But it should be localized at the source, I guess, localization functions should not be used for variables.